frontend/src/components/admin/Tcaption.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { ImSearch } from 'react-icons/im'
+
+type Props = {
+  title: string
+  description: string
+  setSearchedVal: React.Dispatch<React.SetStateAction<string>>
+}
+
+const TCaption: React.FC<Props> = (props) => {
```

Comment:
what does TCaption mean? update filename to something that is easily understandable by other devs